LICENSE.md

```diff
@@ -27,6 +27,10 @@ for exceptions.
 > OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 > WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+Julia includes code from the following projects, which have their own licenses:
+- [LDC](https://github.com/ldc-developers/ldc/blob/master/LICENSE) (for ccall/cfunction ABI definitions)
+- [MUSL](http://git.musl-libc.org/cgit/musl/tree/COPYRIGHT) (for getopt implementations on Windows)
+- [NetBSD](http://www.netbsd.org/about/redistribution.html) (for setjmp/longjmp implementations on Windows)
```

Comment:
The LDC license concerns me a bit. It's not a common license and it's not simple either. The main MUSL license is MIT, same as Julia; is the getopt implementation under that license or a different one? Similarly, is the NetBSD setjmp/longjmp code on Windows under the 2 or 3 clause BSD license? The 2-clause is MIT-equivalent, while the 3-clause is slightly more restrictive (but not much).